lib/plugins/inventory.js

```diff
@@ -359,9 +359,9 @@ function inject (bot, { version, hideErrors }) {
 
     function onRejected () {
       bot._client.write('transaction', {
-        windowId: 0,
+        windowId: click.windowId,
         action: click.id,
-        accepted: false
+        accepted: true
```

Comment:
why accepted true ?

Reply:
Accepted true was the value used by the vanilla 1.16.5 client when sending apology packets. (this was determined using [pakkit](https://github.com/Heath123/pakkit))